cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Fatalf("cmd.StdoutPipe: got %v, want nil", err)
```

Comment:
Not really an issue, but you could stand to have something more descriptive than just cmd.StdoutPipe. Something like "Failed to get stdout of yes command: got %v ..." instead.

Reply:
that's my fault, sorry. Sure make it more verbose.

Reply:
Fixed it :)!